services/catarse/spec/actions/billing/payments/create_spec.rb

```diff
@@ -37,6 +37,41 @@
 
     it { is_expected.to be_success }
 
+    context `when shipping address isn't present` do
+      it 'returns equal attributes' do
+        original_address = ::Shared::Address.find(attributes[:billing_address_id])
+        address_replica = ::Shared::Address.where.not(id: original_address.id).find(result.payment.billing_address_id)
```

Comment:
**Sugestão**
Não precisa do `::` no início desse `::Share::Address` pois tu não está dentro de um móduto.
```suggestion
        original_address = Shared::Address.find(attributes[:billing_address_id])
        address_replica = Shared::Address.where.not(id: original_address.id).find(result.payment.billing_address_id)
```